channels/jaicp/README.md

```diff
@@ -37,24 +37,18 @@ implementation("ch.qos.logback:logback-classic:1.2.3")
 
 ![Create first project in JAICP](https://i.imgur.com/5r35CCv.gif)
 
-#### 3. Create suitable `JaicpWebhookConnector` or `JaicpPollingConnector` to connect your bot to JAICP infrastructure
-Webhook can be created using [Ktor](https://ktor.io) or [Spring Boot](https://spring.io/projects/spring-boot). Here is Ktor implementation example:
+#### 3. Create suitable `JaicpServer` or `JaicpPollingConnector` to connect your bot to JAICP infrastructure
```

Comment:
It should use `JaicpWebhookConnector`, because `JaicpServer` contains Ktor bindings. 
Since last review I've added `JaicpServlet` class for Spring Boot.